frontend/src/app/pages/creator/creator.component.html

```diff
@@ -23,22 +23,18 @@ <h2>{{ user.nickname }}</h2>
       </div>
     </div>   
   </div>
-  <p-tabView>
-    <ul *ngIf="videos">
-      <div class="container">
-        <div class="row" *ngFor="let row of getRows()">
-          <div class="column" *ngFor="let video of getListForRow(row)">              
-            <div class="image-container">
-              <img [src] = "video.thumbnail" class="video">
-              <span class="image-label">{{ video.duration }}</span>
-            </div>
-            <h3 class="noPadding">{{ video.title }}</h3>
-            <p class="noPadding" (click)="goToUserProfile(video.authorId)">{{ video.authorNickname }}</p>
-            <p class="noPadding">{{ video.viewCount }} views</p>
-          </div>
+  <ul *ngIf="videos">
```

Comment:
what if it's empty?

Reply:
![image](https://user-images.githubusercontent.com/95054664/229616630-28826a70-0a4b-45cc-bb66-ce1a98402fe0.png)